doc/user/external.xml

```diff
@@ -306,54 +306,90 @@
-            your build/debug/compile iterations.
+            Ninja is a small build system that tries to be fast
+            by not making decisions. &SCons; can at times be slow
+            because it makes lots of decisions to carry out its goal
+            of "correctness".  The two tools can be paired to benefit
+            some build scenarios: by using the &t-link-ninja; tool,
+            &SCons; can generate the build file &ninja; uses (basically
+            doing the decision-making ahead of time and recording that
+            for &ninja;), and can invoke &ninja; to perform a build.
+            For situations where relationships are not changing, such
+            as edit/build/debug iterations, this works fine and should
+            provide considerable speedups for more complex builds.
+            The implication is if there are larger changes taking place,
+            &ninja; is not as appropriate - but you can always use &SCons;
+            to regenerate the build file.  You are NOT advised to use
+            this for production builds.
+        </para>
+
+        <para>
+            To use the &t-link-ninja; tool you'll need to first install the
+            &Python; &ninja; package, as the tool depends on being able to do an
+            <systemitem>import</systemitem> of the package.
+            This can be done via:
         </para>
+
+        <example_commands>
+# In a virtualenv, or "python" is the native executable:
+python -m pip install ninja
+
+# Windows using Python launcher:
+py -m pip install ninja
+
+# Anaconda:
+conda install -c conda-forge ninja
+        </example_commands>
+
         <para>
-            It's not expected that the ninja builder will work for all builds at this point. It's still under active
-            development. If you find that your build doesn't work with ninja please bring this to the users mailing list
-            or devel channel on our discord server.
+            Reminder that like any non-default tool, you need to initialize it before use
+            (e.g. <code>env.Tool('ninja')</code>).
         </para>
+
         <para>
-            Specifically if your build has many (or even any) python function actions you may find that the ninja build
-            will be slower as it will run ninja, which will then run SCons for each target created by a python action.
-            To alleviate some of these, especially those python based actions built into SCons there is special logic to
-            implement those actions via shell commands in the ninja build file.
+            It is not expected that the &b-link-Ninja; builder will work for all builds at this point. It is still under active
+            development. If you find that your build doesn't work with &ninja; please bring this to the users mailing list
+            or devel channel on our Discord server.
```

Comment:
`#scons-help `channel probably better. Unless they're going to help fix it.

Reply:
Okay, updated.